types/karma/karma-tests.ts

```diff
@@ -248,4 +249,18 @@ karma.constants.COLOR_PATTERN; // $ExpectType string
 karma.constants.NO_COLOR_PATTERN; // $ExpectType string
 karma.constants.CONSOLE_APPENDER; // $ExpectType { type: string; layout: { type: string; pattern: string; }; }
 karma.constants.EXIT_CODE; // $ExpectType string
-karma.constants.LOG_PRIORITIES[5] === "DEBUG";
+karma.constants.LOG_PRIORITIES[5] === 'DEBUG';
+
+// documented api test
```

Comment:
nit: IMO this is unnecessary as there is already test code covering `parseConfig` and `Server` usage.

Reply:
thanks, amended!